### PR TITLE
docs: Adding darkness and brightness examples

### PR DESCRIPTION
**Pull request by erickzanardo**
# Description

Adds two additional examples on the image stories for brightness and darkness changes on images.

https://user-images.githubusercontent.com/835641/229366147-df4701df-901c-4d50-827e-01b57c9f57f9.mov

## Checklist

- [x] I have followed the [Contributor Guide] when preparing my PR.
- [x] I have updated/added tests for ALL new/updated/fixed functionality.
- [x] I have updated/added relevant documentation in `docs` and added dartdoc comments with `///`.
- [x] I have updated/added relevant examples in `examples` or `docs`.

## Breaking Change?

- [ ] Yes, this PR is a breaking change.
- [x] No, this PR is not a breaking change.

## Related Issues

[Contributor Guide]: https://github.com/flame-engine/flame/blob/main/CONTRIBUTING.md
[Conventional Commit]: https://conventionalcommits.org
[CHANGELOG]: https://github.com/flame-engine/flame/blob/main/CHANGELOG.md

### CODE REVIEW

File: examples/lib/stories/image/brighten.dart [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+  static const String description = '''
+     Shows how a dart:ui `Image` can be brightened using Flame Image extensions.
+     Use the properties on the side to change the brightness of the image.
+  ''';
+
+  @override
+  Future<void> onLoad() async {
+    final image = await images.load('flame.png');
+    final brightenImageimage = await image.brighten(brightness / 100);
```
Comment (spydon) [inline examples/lib/stories/image/brighten.dart:20]:
```suggestion
    final brightenedImage = await image.brighten(brightness / 100);
```

File: examples/lib/stories/image/brighten.dart [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+     Shows how a dart:ui `Image` can be brightened using Flame Image extensions.
+     Use the properties on the side to change the brightness of the image.
+  ''';
+
+  @override
+  Future<void> onLoad() async {
+    final image = await images.load('flame.png');
+    final brightenedImage = await image.brighten(brightness / 100);
+
+    add(
```
Comment (luanpotter) [inline examples/lib/stories/image/brighten.dart:22]: did we ever impl that Row/Column Component idea?

**Review by luanpotter — APPROVED** [review]
Nice!

File: examples/lib/stories/image/brighten.dart (hunk 2 above)
Comment (spydon) [reply examples/lib/stories/image/brighten.dart:22]: Only this one, and the Row/Column components should follow suit:
https://github.com/flame-engine/flame/pull/2350

**Review by renancaraujo — APPROVED** [review]

**Comment by spydon** [comment]
The blue artifact is not there anymore, but it still produces a white rectangle on the transparent background.

**Comment by spydon** [comment]
> @spydon ins't the white rectangle expected? I remember we discussed something about it internally?

That was because previously `flame.png` had a black background, but it has a transparent background now and a fully transparent color shouldn't be brightened? Maybe that is actually a bug in our code?

**Comment by spydon** [comment]
> Hi, I randomly found this pr here and I like the image example here. It would be nice if this could be merged so it can also be found easily on the website. What's missing? Can I help here?

As you can see in the video it wasn't working like intended. If you want to help you can checkout the PR, run the example and see if it works on the latest version of Flutter. :)

**Comment by spydon** [comment]
Commits from here added in #3414! :partying_face:

**Pull request closed without merging**